src/main/resources/templates/mdBenefitsFlow/householdMoneyOnHand.html

```diff
@@ -22,27 +21,12 @@
                 <th:block th:ref="moneyOnHand">
                   <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option1}, label=#{moneyonhand-types.option1})}"/>
+                      th:each="option : ${T(org.mdbenefits.app.data.enums.MoneyOnHandType).values()}">
+                    <th:block
+                        th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=${option.getValue()}, label=#{${option.getLabelSrc()}})}"/>
+                  </th:block>
                   <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option2}, label=#{moneyonhand-types.option2})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option3}, label=#{moneyonhand-types.option3})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option4}, label=#{moneyonhand-types.option4})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option5}, label=#{moneyonhand-types.option5})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option6}, label=#{moneyonhand-types.option6})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option7}, label=#{moneyonhand-types.option7})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option8}, label=#{moneyonhand-types.option8})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option9}, label=#{moneyonhand-types.option9})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value=#{moneyonhand-types.option10}, label=#{moneyonhand-types.option10})}"/>
-                  <th:block
-                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value='None', label=#{moneyonhand-types.option11}, noneOfTheAbove=true)}"/>
+                      th:replace="~{fragments/inputs/checkboxInSet :: checkboxInSet(inputName='moneyOnHandTypes',value='None', label=#{general.none-of-the-above}, noneOfTheAbove=true)}"/>
```

Comment:
I used "NONE" for the value in the additional income pages. "None" is fine here, but we should be maybe try to pick one. And we should make sure our alg that checks them compares with no case.

Reply:
Yes. We should definitely pick one. 

I am happy to go with whichever option. In the basic array condition check we use "None" but it's easy to change: https://github.com/codeforamerica/mdpilot/blob/af5064339c1ba23ba092c1e32db42875b5934681/src/main/java/org/mdbenefits/app/submission/conditions/BasicArrayCondition.java#L20

I will use a message value instead of a string in this case and set it to NONE for consistency with your recent code.

Reply:
Yes - By this, do you mean that we should have a constant that we use to both define "None/NONE"?

Reply:
Sorry, I think I got confused between this comment and the other one that was about an enum. This one is just a string, not an enum, so my suggestion makes no sense.

I feel like trying to describe the way I would write this would just be confusing. Can I send a suggested code change after you merge this?